docs/src/main/sphinx/connector/snowflake.md

```diff
@@ -49,30 +49,156 @@ multiple instances of the Snowflake connector.
 
 ## Type mapping
 
-Trino supports the following Snowflake data types:
-
-| Snowflake Type | Trino Type     |
-| -------------- | -------------- |
-| `boolean`      | `boolean`      |
-| `tinyint`      | `bigint`       |
-| `smallint`     | `bigint`       |
-| `byteint`      | `bigint`       |
-| `int`          | `bigint`       |
-| `integer`      | `bigint`       |
-| `bigint`       | `bigint`       |
-| `float`        | `real`         |
-| `real`         | `real`         |
-| `double`       | `double`       |
-| `decimal`      | `decimal(P,S)` |
-| `varchar(n)`   | `varchar(n)`   |
-| `char(n)`      | `varchar(n)`   |
-| `binary(n)`    | `varbinary`    |
-| `varbinary`    | `varbinary`    |
-| `date`         | `date`         |
-| `time`         | `time`         |
-| `timestampntz` | `timestamp`    |
-
-Complete list of [Snowflake data types](https://docs.snowflake.com/en/sql-reference/intro-summary-data-types.html).
+Because Trino and Snowflake each support types that the other does not, this
+connector {ref}`modifies some types <type-mapping-overview>` when reading or
+writing data. Data types may not map the same way in both directions between
+Trino and the data source. Refer to the following sections for type mapping in
+each direction.
+
+List of [Snowflake data types](https://docs.snowflake.com/en/sql-reference/intro-summary-data-types.html).
+
+### Snowflake type to Trino type mapping
+
+The connector maps Snowflake types to the corresponding Trino types following
+this table:
+
+
+:::{list-table} Snowflake type to Trino type mapping
+:widths: 30, 30, 40
+:header-rows: 1
+
+* - Snowflake type
+  - Trino type
+  - Notes
+* - `NUMBER`
+  - `DECIMAL`
+  - Default precision and scale are (38,0).
+* - `DECIMAL`, `NUMERIC`
+  - `DECIMAL`
+  - Synonymous with `NUMBER`. See Snowflake
```

Comment:
can you wrap all these where possible (leave link as long lines)

Reply:
Ack. Updated.